src/plugins/filters/components/FiltersView.vue

```diff
@@ -90,14 +90,20 @@ export default {
         this.composition.load();
         this.unobserve = this.openmct.objects.observe(this.providedObject, 'configuration.filters', this.updatePersistedFilters);
         this.unobserveGlobalFilters = this.openmct.objects.observe(this.providedObject, 'configuration.globalFilters', this.updateGlobalFilters);
-        this.unobserveAllMutation = this.openmct.objects.observe(this.providedObject, '*', (mutatedObject) => this.providedObject = mutatedObject);
+        if (this.providedObject.isMutable !== true) {
```

Comment:
This * observer should not be necessary at all. The selection API will provide `MutableDomainObject`s where appropriate, and if its not mutable, then it shouldn't ever change.